section10/README.md

```diff
@@ -0,0 +1,3 @@
+# Section 10: Acceptance testing
+
+- On Windows can still use the PyCharm Multirun plugin, or now PyCharm CE comes with a "Compound" run configuration which you can use to run two other run configurations sequentially.
```

Comment:
This sentence is a bit hard to understand. What do you think about this (made by ChatGPT-3.5):

```suggestion
- On Windows, you have the option to utilize the PyCharm Multirun plugin, or alternatively, PyCharm CE now offers a "Compound" run configuration, allowing you to execute two other run configurations consecutively.
```

Reply:
Just by seeing the word "utilize" you can tell it's ChatGPT 😂

I do wonder why not just "use" 🤔